Settings.hs

```diff
@@ -80,6 +83,7 @@ instance FromJSON AppSettings where
         appCopyright              <- o .: "copyright"
         appAnalytics              <- o .:? "analytics"
 
+        appDevelopment            <- o .:? "development"      .!= defaultDev
```

Comment:
in that case, should we remove [this](https://github.com/yesodweb/yesod-scaffold/blob/304a23c779fd43bf789662ab49d839784a6d86ac/config/settings.yml#L16) commented out variable?